### PR TITLE
Add auto's all-contributors plugin

### LINKED ISSUE #16: add auto's all contributors plugin
Auto: https://github.com/intuit/auto
Plugin: https://github.com/intuit/auto/tree/master/plugins/all-contributors

### PR DESCRIPTION
**Pull request by rajatdiptabiswas**
# What Changed
- Add all-contributors plugin using `yarn add -D @auto-it/all-contributors`.
- Use `npx all-contributors init` to initialize plugin.
- Change plugin badge position in `README.md` to look more like `intuit/auto` repository.

# Why
Fixes #16 

Todo:
- [ ] Add Semantic Version Label 
- [ ] Add tests
- [ ] Add docs

<details>
  <summary>📦 Published PR as canary version: <code>0.11.8-canary.18.230.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install eslint-plugin-no-explicit-type-exports@0.11.8-canary.18.230.0
  # or 
  yarn add eslint-plugin-no-explicit-type-exports@0.11.8-canary.18.230.0
  ```
</details>

### CODE REVIEW

**Comment by rajatdiptabiswas** [comment]
Hi @kendallgassner 

I am not exactly sure if I managed to make all the required changes.

Kindly take a look and let me know if you need something changed.
I'll try to get them fixed.

I used the following settings to set up `npx all-contributors init`.
Screen Shot 2020-10-01 at 2 09 54 AM

Thanks!

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "projectOwner": "intuit",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": true,
+  "commitConvention": "none",
+  "contributors": [],
```
Comment (kendallgassner) [inline .all-contributorsrc:12]:
```suggestion
  "contributors": [  "contributors": [
    {
      "login": "kendallgassner",
      "name": "Kendall Gassner",
      "avatar_url": "https://avatars3.githubusercontent.com/u/15275462?v=4",
      "profile": "https://github.com/kendallgassner",
      "contributions": [
        "code",
        "doc",
        "ideas"
      ]
    },
    {
      "login": "hipstersmoothie",
      "name": "Andrew Lisowski",
      "avatar_url": "https://avatars3.githubusercontent.com/u/1192452?v=4",
      "profile": "https://github.intuit.com/hipstersmoothie",
      "contributions": [
        "doc"
      ]
    },
    {
      "login": "Rashmi-K-A",
      "name": "Rashmi K A",
      "avatar_url": "https://avatars2.githubusercontent.com/u/39820442?v=4",
      "profile": "https://github.intuit.com/Rashmi-K-A",
      "contributions": [
        "code"
      ]
    },
  ],
```

Comment (kendallgassner) [reply .all-contributorsrc:12]: ^ we need to backfill old contributors

Comment (rajatdiptabiswas) [reply .all-contributorsrc:12]: Sure. I'll make the above changes.

**Comment by hipstersmoothie** [comment]
The plugin still needs to be added to the auto config https://intuit.github.io/auto/docs/generated/all-contributors#usage

**Comment by rajatdiptabiswas** [comment]
Hi @hipstersmoothie 

Do I simply add the following line to `.autorc`?
```
{
  "plugins": ["all-contributors"]
}
```

Is this correct? 
I'll make the changes as soon as you give the green light.

**Comment by kendallgassner** [comment]
yes! @rajatdiptabiswas  .autorc is correct

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "eslint": ">= 6.x"
   },
   "devDependencies": {
+    "@auto-it/all-contributors": "^9.53.1",
```
Comment (hipstersmoothie) [inline package.json:36]: Can you update the version of `auto` to match this too?

File: .autorc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["all-contributors"]
```
Comment (hipstersmoothie) [inline .autorc:2]:
```suggestion
  "plugins": ["npm", "all-contributors"]
```

**Review by hipstersmoothie — CHANGES_REQUESTED** [review]

File: .autorc (hunk 3 above)
Comment (rajatdiptabiswas) [reply .autorc:2]: No worries. I'm looking into them.

File: package.json (hunk 2 above)
Comment (rajatdiptabiswas) [reply package.json:36]: Hi @hipstersmoothie 

I am sorry but I am having trouble understanding what I need to do here.
Do I run `yarn upgrade` or `npm update` on `auto`?

Running `yarn outdated` showed that there is a much newer version of `auto` available.

Kindly let me know what needs to be done.
Thanks!

Comment (kendallgassner) [reply package.json:36]: Can you bump both to the *same* version

Comment (rajatdiptabiswas) [reply package.json:36]: Hello @kendallgassner 
Does this mean the required change is as follows?

In `package.json`
```
"devDependencies": {
    "@auto-it/all-contributors": "^9.53.1",
    ...
    "auto": "^9.53.1",
    ...
}
```

Comment (kendallgassner) [reply package.json:36]: yes! thats perfect

**Comment by kendallgassner** [comment]
Can you run yarn install and add in the yarn.lock file to the commit

**Comment by rajatdiptabiswas** [comment]
Sure @kendallgassner 
Doing it right away!

**Comment by kendallgassner** [comment]
okay this is the last thing i think 

in Package.json the 'release' script uses the flag '-w' this flag was deprecated in the new version of auto ... can you replace it with '-vv'

**Comment by rajatdiptabiswas** [comment]
I'm sorry for not seeing this earlier @kendallgassner.
I'll get it done right now.

**Review by kendallgassner — APPROVED** [review]

**Comment by rajatdiptabiswas** [comment]
Thanks @kendallgassner for guiding me through this!

**Pull request merged**